test/chainsaw/integration/app-one-taskdefinition-not-found/chainsaw-test.yaml

```diff
@@ -0,0 +1,19 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/kyverno/chainsaw/main/.schemas/json/test-chainsaw-v1alpha1.json
+apiVersion: chainsaw.kyverno.io/v1alpha1
+kind: Test
+metadata:
+  name: app-one-taskdefinition-not-found
+spec:
+  steps:
+    - name: step-00
+      try:
+        - script:
+            content: kubectl annotate ns $NAMESPACE keptn.sh/lifecycle-toolkit='enabled'
+        - apply:
+            file: 00-install.yaml
+        - assert:
+            file: 00-assert.yaml
+    - name: step-01
+      try:
+        - script:
+            content: ../../common-scripts/verify-number-of-resources.sh "keptntask" "1" $NAMESPACE
```

Comment:
@eddycharly this could be done with assertions on the kubectl result right? 

somehting on the line of
```
  - script:
        content: kubectl get TaskCRD in namespace
        check:
          # in stdout or else fails
          (contains($stdout, 1)): true
```

If yes I will add it to our fixes ticket :)

Reply:
Once everything is converted i will go through all commands/scripts and see if they can be removed anyway.
I already have one or two features i could add to chainsaw for that :)

Reply:
Anyway tracking this with an issue is a good idea.